interface/forms/patient_encounter/save.php

```diff
@@ -146,7 +146,26 @@
 ?>
 <html>
 <body>
+<?php
+	require_once("../../globals.php");
+	require_once("$srcdir/headers.inc.php");
+?>
+
+<?php call_required_libraries(array("jquery-min-3-1-1" , "font-awesome", "iziModalToast")); ?>
+
 <script language='JavaScript'>
+
+    iziToast.success({
+        title: 'Success -',
+        message: "Successfully added new encounter",
+        position: 'center',
+        icon: 'fa fa-thumbs-o-up'
+
+    });
+    
+    setTimeout(function () {
+        parent.$("#izi-iframe").iziModal('close');
+    }, 2000);
```

Comment:
@muarachmann Think about setting a global for the length of time on the messages. Experienced users may want them to disappear faster new installs may need to read them.